src/main/java/Main.java

```diff
@@ -19,12 +19,22 @@ public static void main(String[] args){
             if (guess.equals(previous)){
                 System.out.println("You just guessed that!");
                 //we're only checking for the previous guess, not all previous guesses, so we dont need a dictionary
-            } else {
+            } else if (guess > target){
                 guessCounter += 1;
                 previous = guess;
+                System.out.println("Too large! Guess again.");
+                //iterate guess counter, change previous to our new guess, go back to our loop
+            } else if (guess < target){
+                guessCounter += 1;
+                previous = guess;
+                System.out.println("Too small! Guess again.");
                 //iterate guess counter, change previous to our new guess, go back to our loop
```

Comment:
what if the guess is too high? please rework.

Reply:
I'm confused, the else if on line 22 should be handling that case, and incrementing the int guessCounter the same as the else if shown here. Is there something I've missed?